tests/visualization_tests/test_edf.py

```diff
@@ -28,27 +39,39 @@ def test_plot_optimization_history(direction: str) -> None:
     study0 = create_study(direction=direction)
     study0.optimize(lambda t: t.suggest_float("x", 0, 5), n_trials=10)
     figure = plot_edf(study0)
+    data = figure.data
+    _validate(data[0]["y"])
     assert len(figure.data) == 1
     assert figure.layout.xaxis.title.text == "Objective Value"
 
     # Test with two studies.
     study1 = create_study(direction=direction)
     study1.optimize(lambda t: t.suggest_float("x", 0, 5), n_trials=10)
     figure = plot_edf([study0, study1])
+    data = figure.data
+    for tmp_data in data:
+        _validate(tmp_data["y"])
```

Comment:
ditto. In addition, `tmp_data` may be a bit ambiguous. Can we rename it?
```suggestion
    for points in figure.data:
        _validate(points["y"])
```